app/assets/javascripts/application.js

```diff
@@ -12,5 +12,6 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require bootstrap
```

Comment:
This is not really related to this feature and should be committed into other PR. If we ever need to revert this to use another frontend library, we would be obligated to also revert the OAuth feature. But that's ok for now, we can go on like this. Just a heads up for good practice in code committing.